aws/cloudformation/cloud_formation_stack.yml.erb

```diff
@@ -158,29 +226,20 @@ Resources:
             VolumeSize: 64
             VolumeType: gp2
       UserData:
-        Fn::Base64: <%=file.call('bootstrap_chef_stack.sh.erb',
-          resource_id: 'WebServer',
+        Fn::Base64: <%=file.call('bootstrap_frontend.sh.erb',
           node_name: 'fe-${INSTANCE_ID}',
-          run_list: ['role[front-end]'],
-          commit: commit )%>
-  WebServerHook:
-    Type: AWS::AutoScaling::LifecycleHook
-    Properties:
-      AutoScalingGroupName: {Ref: WebServer}
-      LifecycleTransition: 'autoscaling:EC2_INSTANCE_LAUNCHING'
-      DefaultResult: ABANDON
-      HeartbeatTimeout: 3000 # seconds = 50 minutes
-      NotificationTargetARN: {Ref: WebServerHookTopic}
-      RoleARN: {'Fn::GetAtt': [IAM, LifecycleHookRoleARN]}
-  WebServerHookTopic: {Type: 'AWS::SNS::Topic'}
```

Comment:
We no longer need a LifecycleHook because the AMI the frontends use is already configured with the correct code revision, so as soon as it boots up and starts serving traffic it can be put in service without waiting.